validation-test/Sema/type_checker_perf/slow/intinit.swift.gyb

```diff
@@ -0,0 +1,10 @@
+// RUN: not %scale-test --begin 1 --end 4 --step 1 %s
```

Comment:
Apparently this is going to do three runs so it might make sense to add a few more so that there is more data to fit to. @graydon found that he had to do this with several of the tests that I added in order to get his new curve fitting code to work.

Reply:
I tried more, but with `--end 5` it starts failing with "expression too complex" 🙃

Reply:
Ah yeah, that's been known to happen. 😐